pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -1536,6 +1546,31 @@ public MetricsSystem getMetricsSystem() {
     return StaticNodesParser.fromPath(staticNodesPath);
   }
 
+  private Function<Long, Long> gasLimitCalculator() {
```

Comment:
This function should live somewhere else, not in the class configuring stuff based off of the CLI.  Perhaps in the class I mention in PantheonContextBuilder.